src/payment/CardHolderInformation.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Field } from 'redux-form';
+import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-i18n';
+
+import { countryOptionsSelector } from './data/selectors';
+import messages from './CardHolderInformation.messages';
+
+class CardHolderInformation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.countryOptions = [{
```

Comment:
One thing to note is that by doing this in the constructor, it'll only happen once when the CardHolderInformation component is instantiated, prior to any lifecycle methods.  If the countryOptions prop is updated/gets touched for whatever reason, it'll lose this added value.  Doing this in renderCountryOptions would ensure it always happens, and would remove the need for the instance variable.

Reply:
Sounds good. This was a copy/paste from one of the other apps.